src/cobalt_model/frontmatter.rs

```diff
@@ -214,6 +227,13 @@ impl FrontmatterBuilder {
         self.merge(Self::new().set_published_date(published_date.into()))
     }
 
+    pub fn merge_pagination<S: Into<Option<pagination_config::PaginationConfigBuilder>>>(
+        self,
+        secondary: S,
+    ) -> Self {
+        self.merge(Self::new().set_pagination(secondary.into()))
```

Comment:
Like `merge_data`, should this call `merge_pagination`?

Reply:
At first I commented on this, thinking extra processing is needed for deep members but I'm not even sure `merge_data` needs to be special cased.

Reply:
So nothing to be done here in the end?

Reply:
Yeah, this should be fine for now.